cinch/roles/nginx/handlers/main.yml

```diff
@@ -0,0 +1,8 @@
+- name: reload systemd
```

Comment:
Reading the Ansible docs on this feature indicates (at least to me) that it's supported in 2.2:

https://docs.ansible.com/ansible/systemd_module.html

Am I missing something there?  Now there is a note at the bottom about special conditions for 2.4, but those appear to be some required options, but I don't think that prevents us from using the module in 2.2.